src/variables.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+from eth_account import Account
+
+# - Providers-
+EXECUTION_CLIENT_URI = os.getenv('EXECUTION_CLIENT_URI', '').split(',')
+CONSENSUS_CLIENT_URI = os.getenv('CONSENSUS_CLIENT_URI', '')
```

Comment:
Can we support fallback CL nodes like in EL?

Reply:
https://www.notion.so/Can-we-support-fallback-CL-nodes-like-in-EL-8f801ce5b4764f46bc7b120a8bfccc72?pvs=4